### ISSUE TITLE
MacOS Support

### ISSUE THREAD

**Issue by SimonIT** [rekire/autologin_plugin#11]
The [Keychain Service](https://developer.apple.com/documentation/security/keychain_services) can be used on MacOS.

Tutorials for [storing](https://developer.apple.com/documentation/security/keychain_services/keychain_items/adding_a_password_to_the_keychain) and [searching](https://developer.apple.com/documentation/security/keychain_services/keychain_items/searching_for_keychain_items) keychain items.

(I'm adding this just for completeness)

**Comment by rekire** [comment]
I cannot use or test this feature currently, since the App where I'm using this plugin requires the webview which is not supported yet. (And it seems that it will take a lot of time until this will happen)

**Comment by rekire** [comment]
Small update here: I finished that stuff I had to finish first and now I can make further steps. The webview support is still not done (even if it was announced for the end of 2023). In the meantime I learned a lot about MacOS development with Flutter. I will use the recomented APIs a bit mixed with [shared web credentials](https://medium.com/pinterest-engineering/shared-web-credentials-a-simpler-way-to-log-in-f94bca0d683b). This will require some API changes I'll mention in a new ticket more or less as roadmap.
